CGPs/cgp-0076.md

```diff
@@ -0,0 +1,155 @@
+---
+cgp: 76
+title: LatAm-W3-DAO-Proposal
+date-created: 2023-03-25
+author: '@juancamp1987 in behalf of Mila Rioja'
+status: DRAFT
+discussions-to: https://forum.celo.org/t/latam-w3-dao-proposal/4494
+governance-proposal-id: TBD
+date-executed: TBD
+---
+<!---
+YAML ends
+-->
+
+<!---
+Beginnning of the proposal
+-->
+# Abstract
+To spread awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish! 3), and help nurture the next cohort of great initiatives on Web3.
+Celo has one of the most global communities in crypto. 
+In order to further expand decentralization and the localization of the protocol in different regions and cultures, we would love to empower local communities to lead local growth.
+We believe that folks on the ground are the most equipped to do this and that a grassroots approach will benefit the growth of Celo and its community.
+
+
+# Proposal
+
+## Goal
+Bring to life next gen of dApps on Celo, and contribute with the community in projects towards raising awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish!),
```

Comment:
Bring to life next gen of dApps and partnerships to Celo by exploring #REFI use cases and opportunities, and contribute with the community in projects towards raising awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish!).